webapp-django/crashstats/api/tests/test_views.py

```diff
@@ -702,8 +700,12 @@ def test_bad_uuid(self, client):
         data = json.loads(resp.content)
         assert data == {"error": "unknown crash id"}
 
-    @mock_s3_deprecated
-    def test_elastcsearch_has_crash(self, client):
+    def test_elastcsearch_has_crash(self, boto_helper, client):
+        bucket = settings.SOCORRO_CONFIG["resource"]["boto"]["bucket_name"]
+        boto_helper.create_bucket(bucket)
+        telemetry_bucket = settings.SOCORRO_CONFIG["telemetrydata"]["bucket_name"]
+        boto_helper.create_bucket(telemetry_bucket)
```

Comment:
Because we're not using moto and mocking out all s3 things, we need to create the buckets so that they don't throw a "bucket isn't there" error.

Reply:
I should centralize this.